src/dash.ts

```diff
@@ -53,11 +53,6 @@ export class Dash {
    * @return {string} OS-specific URI to pass to handler, mainly because of Windows
    */
   getOSSpecificURI(uri: string): string {
-    return (
-      {
-        // on Windows, start can't deal with the double quotes, and & needs to be escaped with ^
-        win32: uri.replace('&', '^&'),
-      }[this.OS] || '"' + uri + '"'
-    );
+    return ({}[this.OS] || '"' + uri + '"');
```

Comment:
should we simplify with this instead?
```
return '"' + uri + '"';
```

Reply:
Either way works. I kept it open like this just in case any OS-specific formatting is needed again for some reason.

Reply:
If I were really going all-out into my own personal coding style, and didn't want to keep the *(currently no-op)* switch expression, I'd probably use backticks:
```js
return `"${uri}"`;
```

Reply:
_(... especially since the eyes start to blur on the adjacent single/double quotes... )_

Reply:
yes, using your solution to use string interpolation is much better.  👍please change it  

for OS specific, let's just remove it since we don't use it anymore, we can just refer to it in git history eventually if we need it again.

Reply:
Done and done.

Reply:
Simplified the code a bit, then looked at the new merge conflicts and changed it to be compatible with the new "exact" options.